sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -142,11 +152,11 @@ When working with JSON from Azure services, you can learn what properties are av
 
 Note that most Azure services name JSON fields [with camelCase names](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#json-field-name-casing) to [treat them with case-sensitivity](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#json-field-names-case-sensitivity).  Not every Azure service adheres to this convention; please consult the service REST API documentation.
 
-If you are using the `DynamicCaseMapping.PascalToCamel` setting and there is a need to bypass these name mappings, JSON members can be accessed with exact strings using property indexers.
+If you set `ClientOptions.Protocol.PropertyNamingConvention` to a value other than `PropertyNamingConvention.None` and there is a need to bypass the name mapping, JSON members can be accessed with exact strings using property indexers.
```

Comment:
Couldn't you also just pass CaseMapping.None into the ToDynamicFromJson call?

Reply:
Cool, that's a small addition, happy to take it.

Reply:
And nice spot, too, btw, I didn't see that, but I think it means this works better than I'd expected :)

Reply:
Updated.